contents/docs/experiments/holdouts.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Holdouts
+---
+
+Holdouts let you create persistent groups of users who can be excluded from multiple experiments or all experiments.
+
+To create a new holdout, navigate to "Experiments" > "Holdout groups" and click "New holdout". Enter a descriptive name for your holdout, fill out the optional description to help you remember its purpose, and set the percentage of users you want to include in the holdout. We recommend between 1% and 10%; the holdout should be large enough to be statistically significant, but small enough as to not significantly reduce the pool of users available for experiments.
```

Comment:
Let's also add a screenshot here. Use <ProductScreenshot> to upload both dark and light modes
```suggestion
To create a new holdout:

1. Click the [Holdout groups](add the link to the posthog app here e.g. [https://](https://us.posthog.com)/experiments/holdouts or whatever it is) in the experiments tab".
2. Click **New holdout**. 
3. Enter a descriptive name for your holdout, fill out the optional description to help you remember its purpose, and set the percentage of users you want to include in the holdout. 

The holdout should be large enough to be statistically significant, but small enough as to not significantly reduce the pool of users available for experiments. We recommend between 1% and 10%.
```